libnmstate/netinfo.py

```diff
@@ -33,3 +35,13 @@ def show(*, include_status_data=False):
     """
     with plugin_context() as plugins:
         return show_with_plugins(plugins, include_status_data)
+
+
+def show_running_config():
+    with plugin_context() as plugins:
+        return show_running_config_with_plugins(plugins)
+
+
+def show_saved_config():
```

Comment:
Usually, the configuration is called state in nmstate, so maybe use `state` here, too.

Reply:
`state` indicate full network status. `config` is for configuration only.

Reply:
I am fine with the name, but should it be an argument of show() instead of a new function?

Reply:
The `show()` is returning a state, while this function returning a config. So I prefer `show_saved_config()` over
`show(type=saved)` as the latter one require a constant.

Reply:
@EdDev What's your idea on this? Should we use new argument of `show()` or new function?

Reply:
It's been a while. And we already merged the `show_running_config()` PR #1419 .